src/main/java/frc/robot/util/LogManager.java

```diff
@@ -1,38 +1,111 @@
 package frc.robot.util;
 
-import edu.wpi.first.util.datalog.DataLog;
-import edu.wpi.first.util.datalog.IntegerLogEntry;
-import edu.wpi.first.util.datalog.StringLogEntry;
-import edu.wpi.first.wpilibj.DataLogManager;
-import java.time.Duration;
 import java.util.ArrayList;
 import java.util.function.Supplier;
 
+import dev.doglog.DogLog;
+
 /**
  * Utilty class for logging data to the DataLogManager.
  * View logs using MechanicalAdvantage's advantage scope (<a href="https://github.com/Mechanical-Advantage/AdvantageScope">...</a>)
  */
-public class LogManager {
-
-  public static DataLog DATA_LOG = DataLogManager.getLog();
+public class LogManager extends DogLog {
```

Comment:
The name of the branch is clawLog. Do you plan to leave this as LogManager or rename it to ClawLog?